themes/src/main/resources/META-INF/keycloak-themes.json

```diff
@@ -8,5 +8,8 @@
     }, {
         "name" : "keycloak.v2",
         "types": [ "account", "admin" ]
+    },{
```

Comment:
we need a separate theme for Admin but not urgent

Reply:
ok